controllers/new.go

```diff
@@ -637,7 +637,7 @@ func (k *kubeResources) newGraphBuilderVolumeMounts(instance *cv1.UpdateService)
 		vm = append(vm, corev1.VolumeMount{
 			Name:      NameClusterTrustedCAVolume,
 			ReadOnly:  true,
-			MountPath: "/etc/pki/ca-trust/extracted/pem/cluster-ca-bundle.crt",
+			MountPath: "/etc/pki/ca-trust/extracted/pem/ca-bundle.crt",
```

Comment:
We still use the hard coded path here. Can we use a variable instead as we have the file name in `NameClusterCertConfigMapKey`?